playground/samples/defaults.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+  schema: {
+    title: "Schema default properties",
+    type: "object",
+    properties: {
+      valuesInFormData: {
+        title: "Values in form data",
+        $ref: "#/definitions/defaultsExample",
+      },
+      noValuesInFormData: {
+        title: "No values in form data",
+        $ref: "#/definitions/defaultsExample",
+      },
+    },
+    definitions: {
+      defaultsExample: {
+        type: "object",
+        properties: {
+          scalar: {
+            title: "Scalar",
+            type: "string",
+            default: "scalar default",
+          },
+          array: {
```

Comment:
Can you rename `array` to something else? (it would be clearer in the playground if the array were not named `array`, just because `type: array` is already there)

Also for `object` too.

Reply:
nvm